lib/google_apis/generator.ex

```diff
@@ -14,4 +14,31 @@
 
 defmodule GoogleApis.Generator do
   @callback generate_client(GoogleApis.ApiConfig.t) :: any()
+  alias GoogleApis.ApiConfig
+
+  def bump_version(api_config) do
+    mixfile = Path.join(ApiConfig.library_directory(api_config), "mix.exs")
+
+    new_contents =
+      mixfile
+      |> File.stream!
+      |> Stream.map(&replace_version/1)
+      |> Stream.into([])
+      |> Enum.join("")
+
+    File.write!(mixfile, new_contents)
+  end
+
+  defp replace_version(line) do
+    Regex.replace(~r/@version "(\d+\.\d+\.\d+)"/, line, fn _, version_str -> bump_version_string(version_str) end)
+  end
+  defp bump_version_string(str) do
```

Comment:
Has this been put through `mix format`? I'm kinda surprised about no space between separate functions.